testng-test-osgi/testng-test-osgi-build.gradle.kts

```diff
@@ -29,8 +29,12 @@ dependencies {
     testImplementation("org.ops4j.pax.exam:pax-exam-link-mvn:4.13.5")
     testImplementation("org.ops4j.pax.url:pax-url-aether:2.6.12")
     testImplementation("org.apache.felix:org.apache.felix.framework:7.0.5")
-    testImplementation("ch.qos.logback:logback-core:1.4.11")
-    testImplementation("ch.qos.logback:logback-classic:1.4.11")
+    testImplementation("ch.qos.logback:logback-core:1.2.13") {
+        because("logback 1.3+ depends on slf4j-api 2.0+")
+    }
+    testImplementation("ch.qos.logback:logback-classic:1.2.13") {
+        because("logback 1.3+ depends on slf4j-api 2.0+")
+    }
```

Comment:
Note: there should be dependabot configuration to prevent it from upgrading this dependency to 1.3+

The relevant renovatebot config is

```json
    {
      "matchPackagePrefixes": ["ch.qos.logback"],
      "groupName": "logback",
      "allowedVersions": "< 1.3.0"
    },
```